### DISCUSSION TITLE
Happy New Year 2023 to all BiblioteQ users around the world.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#267] [General]
It's great to have an internationalized software  !

A big Bravo and a huge thank you to textbrowser for all the improvements made in 2022; I wish you and your family a great year 2023.
![2023](https://user-images.githubusercontent.com/56431039/210179943-c1cc3b48-dced-4b05-ad11-6fb4714e9719.jpeg)

**Comment by textbrowser** [comment]
Happy New Year and keep off the Internets! :)

**Reply by meteos77** [reply]
I'm having trouble getting back into computers this year :-)
